Test_Notebook_Sklearn.ipynb

```diff
@@ -18,6 +18,7 @@
    },
```

Comment:
<p>I changed the first line and added a comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jb-ds2020/nf-ds3-capstone-churn-prevention/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>